### ISSUE TITLE
Checksum mismatch mongodb-community@4.2

### ISSUE THREAD

**Issue by devfrey** [mongodb/homebrew-brew#15]
Maybe this is related to #13?

`$ brew install mongodb-community@4.2`

```
==> Installing mongodb-community from mongodb/brew
==> Downloading https://fastdl.mongodb.org/osx/mongodb-macos-x86_64-4.2.0.tgz
######################################################################## 100.0%
Error: An exception occurred within a child process:
  ChecksumMismatchError: SHA256 mismatch
Expected: c7214ee7bda3cf9566e8776a8978706d9827c1b09017e17b66a5a4e0c0731e1f
  Actual: 6aa2e0c348e8abeec7931dced1f85d4bb161ef209c6af317fe530ea11bbac8f0
 Archive: /Users/jeffrey/Library/Caches/Homebrew/downloads/a6696157a9852f392ec6323b4bb697b86312f0c345d390111bd51bb1cbd7e219--mongodb-macos-x86_64-4.2.0.tgz
To retry an incomplete download, remove the file above.
```

**Comment by wildlava** [comment]
That checksum does indeed seem incorrect - does not match the downloaded file's.

**Comment by aa7955** [comment]
Yep, same issue.

**Comment by SaulGarza** [comment]
Same Issue here, I just installed 4.0 for now

**Comment by kciter** [comment]
same problem here

**Comment by jiangfengwhu** [comment]
`brew edit mongodb-community`
change the sha256 value works

**Comment by visuddha** [comment]
> `brew edit mongodb-community`
> change the sha256 value works

Yep that works. But is it safe?

**Comment by chasinglogic** [comment]
Fixed by #17
